lang/gnu-apl/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           select 1.0
+
+name                gnu-apl
+version             1.8
+categories          lang
+platforms           darwin
+license             GPL
+maintainers         {@casouri gmail.com:casouri}
```

Comment:
You probably want to add `openmaintainer` to the end here unless you have good reason not to, as in:

```
maintainers         {@casouri gmail.com:casouri} openmaintainer
```

Reply:
Done.